File: _/2183OS_08_Code/Task 2/solr_query.php
<?php
require_once 'Apache/Solr/Service.php';

$solr = new Apache_Solr_Service("dev.fts-wizards.com", "8983", "/solr");

$response = $solr->search("document", 0, 10, null);

echo "DOCUMENTS:\n";
foreach ($response->response->docs as $doc) {
	echo $doc->id . ":" . $doc->name . "\n";
}
?>

File: _/2183OS_08_Code/Task 3/solr_query.rb
require 'rubygems'
require 'rsolr'

solr = RSolr.connect :url => 'http://dev.fts-wizards.com:8983/solr'

response = solr.get 'select', :params => {
    :q => 'document',
    :start => 0,
    :rows => 10
  }

puts 'DOCUMENTS'
response["response"]["docs"].each{|doc| puts doc["id"] + ":" + doc["name"] }
